services/hooks/src/api.js

```diff
@@ -446,7 +446,7 @@ builder.declare({
 
   await req.authorize({ hookGroupId, hookId });
 
-  const payload = req.body;
+  const payload = { ...req.body, ...{ queryParameters: req.query } };
```

Comment:
```suggestion
  const payload = { ...req.body, queryParameters: req.query };
```

I think this can be simplified a bit, or there are something less obvious happening here?

Reply:
JS is not my strong suit :smile: